_episodes/04-drawing-bitwise.md

```diff
@@ -181,103 +173,74 @@ Here is what our constructed mask looks like:
-> > 		y1 = random.randrange(600)
-> > 		cv2.rectangle(img = image, 
-> > 			pt1 = (x1, y1),
-> > 			pt2 = (x1 + 50, y1 + 50),
-> > 			color = (255, 0, 0),
-> > 			thickness = -1)
-> > 
+> >     x = random.random()
+> >     if x < 0.33:
+> >         rr, cc = skimage.draw.circle(
+> >             random.randrange(600),
+> >             random.randrange(800),
+> >             radius=50,
+> >             shape=image.shape[0:2],
+> >         )
+> >         color = (0, 0, 255)
+> >     elif x < 0.66:
+> >         rr, cc = skimage.draw.line(
+> >             random.randrange(600), random.randrange(800),
+> >             random.randrange(600), random.randrange(800)
+> >         )
+> >         color = (0, 255, 0)
+> >     else:
+> >         rr, cc = skimage.draw.rectangle(
+> >             start=(random.randrange(600), random.randrange(800)),
+> >             extent=(50, 50),
+> >             shape=image.shape[0:2]
+> >         )
+> >         color = (255, 0, 0)
+> >
+> >     image[rr, cc] = color
+> >
 > > # display the results
-> > cv2.namedWindow(winname = "image", flags = cv2.WINDOW_NORMAL)
-> > cv2.imshow(winname = "image", mat = image)
-> > cv2.waitKey(delay = 0)
+> > viewer = ImageViewer(image)
+> > viewer.show()
 > > ~~~
 > > {: .python}
 > {: .solution}
 {: .challenge}
 
-## Bitwise operations
-
-All that remains in the task of using our mask is to apply some 
-*bitwise operations* to merge the mask together with the original image,
-in such a way that the areas with white pixels in the mask are shown, while
-the areas with black pixels in the mask are not shown. Recall that if we are 
-using 24-bit color, each RGB value is represented by eight bits. 
-This is what gives us the decimal range [0, 255] for each value. The smallest
-eight-bit value is all zeros (00000000), and the greatest is (11111111).
-This maximum value is 
+## Image modification
 
-1 × 2<sup>7</sup> + 1 × 2<sup>6</sup> + 1 × 2<sup>5</sup> + 
-1 × 2<sup>4</sup> + 1 × 2<sup>3</sup> + 1 × 2<sup>2</sup> + 
-1 × 2<sup>1</sup> + 1 × 2<sup>0</sup>,
-or 
-
-128 + 64 + 32 + 16 + 8 + 4 + 2 + 1 = 255. 
-
-The reason this matters is that masks operate on these types of numbers, at a 
-bit-by-bit level. The main binary operation we will need is *bitwise and*. In
-this situation, we take two numbers in binary, such as 11010110 and 01001101.
-From those two numbers, we produce a third number by examining each bit in
-succession, performing a logical "and" on each pair to produce a bit for the
-new number. The following *truth table* shows how bitwise and works:
-
-![and truth table](../fig/03-and-truth-table.png)
-
-According to the table, "0 and 0" produces 0, as does "0 and 1" and "1 and 0."
-The only time bitwise and produces 1 is the "1 and 1" case. This "and" 
-operation is applied to each bit in a number, hence the term bitwise and. 
-The next image shows how bitwise and would be applied to 11010110 and 01001101.
-
-![Bitwise and example](../fig/03-bitwise-and.png)
```

Comment:
I like that this change makes the section simpler!

Reply:
:) :+1: cool. Do you have a preference in terms of renaming this episode/lesson? Since _bit-wise_ doesn't cut it anymore, I would go for something like `04-drawing.md`, okay?